src/test/kotlin/step3/CarTest.kt

```diff
@@ -0,0 +1,28 @@
+package step3
+
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+
+class CarTest {
+
+    private lateinit var car: Car
+
+    @BeforeEach
+    fun setUp() {
+        car = Car()
+    }
```

Comment:
중요한 부분은 아니나 불필요한 코드로 보이는데요, 아래처럼 선언하는건 어떨까요? BeforeEach가 없어도 의도한대로 테스트가 구동될 것 같습니다.

```suggestion
    private val car: Car = Car()
```